rest/rest-resources/src/main/java/org/apache/brooklyn/rest/transform/EntityTransformer.java

```diff
@@ -162,7 +163,7 @@ public static EntityConfigSummary entityConfigSummary(Entity entity, ConfigKey<?
             SensorTransformer.addNamedAction(lb, na, entity.getConfig(config), config, entity);
         }
     
-        return entityConfigSummary(config, label, priority, lb.build());
+        return entityConfigSummary(config, label, priority, false, lb.build());
```

Comment:
This will mark all config as invisible in `v1/applications/xxxxxxxx/entities/yyyyyyyy/config`. That's fine as long as the UI know about this caveat. Might be better to make this field nullable and set `null` here?